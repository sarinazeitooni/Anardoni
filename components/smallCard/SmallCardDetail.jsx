import React from "react";
import style from './scss/smallCard.module.scss'
const SmallCardDetail = ({title,detail})=>{
    return(
        <div className={style['card-without-container-detail']}>
            <h4>{title}</h4>
            <p>{detail}</p>
        </div>
    )
}
export default SmallCardDetail