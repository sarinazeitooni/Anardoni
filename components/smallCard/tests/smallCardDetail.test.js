import React from 'react';
import { shallow } from 'enzyme';
import SmallCardDetail from '../SmallCardDetail';
describe("SmallCardDetail component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<SmallCardDetail/>)
        expect(container).toMatchSnapshot();
    })
});