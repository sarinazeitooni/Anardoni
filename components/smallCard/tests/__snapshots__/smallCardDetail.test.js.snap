// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallCardDetail component it should render a div 1`] = `ShallowWrapper {}`;
